### DISCUSSION TITLE
Where to enable risk and coverage views for security tab

### DISCUSSION THREAD

**Discussion by josephjthomas** [community/community#38877] [Enterprise]
### Select Topic Area

Question

### Body

Do I need to enable this somewhere? https://github.blog/changelog/2022-11-10-risk-and-coverage-views-on-the-security-tab-for-organizations-public-beta/

My organization doesn't have a security tab.

**Comment by just-joshing** [comment]
Hi, @josephjthomas!
Do you know if your organization is part of a [GitHub Enterprise plan](https://docs.github.com/en/get-started/onboarding/getting-started-with-github-enterprise-cloud)?
The new Risk and Coverage pages are only available today for GitHub Enterprise plans.
